tools/prepare-release/main.go

```diff
@@ -196,12 +192,6 @@ func createCommit(msg string) {
 }
 
 func createTag(version string) {
-	// check if repo is dirty
-	if isRepoDirty() {
-		fmt.Fprintf(os.Stderr, "repo is dirty when creating a new tag for version %s", version)
-		os.Exit(1)
-	}
-
```

Comment:
@SagarGi "something happened" - lines like this should not be deleted in a tests-only PR.